tools/pkg-helpers/pytorch_pkg_helpers/__main__.py

```diff
@@ -33,11 +33,17 @@ def parse_args() -> argparse.Namespace:
     )
     parser.add_argument(
         "--channel",
-        help="Channel to look in",
+        help="Channel to look in, either pytorch-nightly or pytorch-test",
         choices=["nightly", "test"],
         type=str,
         default=os.getenv("CHANNEL", "nightly"),
     )
+    parser.add_argument(
+        "--custom-channel",
+        help="Custom channel to look in, ignore --channel argument when this is set",
+        type=str,
+        default=os.getenv("CUSTOM_CHANNEL", None),
```

Comment:
Do you expect it to be passed via environment variable? Or via command line option

Reply:
Hi @malfet thanks for taking a look. We would probably use this option by setting environment variables. I've added both the command line option and environment variable so the code looks consistent with other argument definitions. Let me know if you think we should get rid of the new option and just handle the new environment variable.

Reply:
closed this PR for now as we are investigating an issue where `conda search 'pytorch[channel={name}]'` doesn't seems to be working for local file system channel like `file://<path>`